src/chainparams.cpp

```diff
@@ -91,8 +91,8 @@ class CMainParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nStartTime = 1199145601; // January 1, 2008
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nTimeout = 1230767999; // December 31, 2008
 
-        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000e1ab5ec9348e9f4b8eb8154");
-        consensus.defaultAssumeValid = uint256S("0x0000000000000000000f2adce67e49b0b6bdeb9de8b7c3d7e93b21e7fc1e819d"); // 623950
+        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000001533efd8d716a517fe2c5008");
+        consensus.defaultAssumeValid = uint256S("0x0000000000000000000b9d2ec5a352ecba0592946514a92f14319dc2b367fc72"); // 654683
```

Comment:
The mainnet nMinimumChainWork and defaultAssumeValid consensus params match up for me
 ```
$ ./src/bitcoin-cli getblockheader 0000000000000000000b9d2ec5a352ecba0592946514a92f14319dc2b367fc72
{
  "hash": "0000000000000000000b9d2ec5a352ecba0592946514a92f14319dc2b367fc72",
  "height": 654683,
  "chainwork": "00000000000000000000000000000000000000001533efd8d716a517fe2c5008",
}
````